packages/flutter_tools/lib/src/device.dart

```diff
@@ -835,6 +835,7 @@ class DebuggingOptions {
     this.startPaused = false,
     this.disableServiceAuthCodes = false,
     this.disableDds = false,
+    this.dartEntrypointArgs,
```

Comment:
A default value for each would be nice `this.dartEntrypointArgs = const <String>[]`